src/main/java/signup_and_login/UserDBAccess.java

```diff
@@ -0,0 +1,103 @@
+package signup_and_login;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserDBAccess implements UserDBGateway {
+
+    private static final String API_URL = "https://api.sheety.co/78ad1edb28469578058ca4c58c3f478b/larklink/users";
```

Comment:
API url should be an env variable, not hard-coded into the class. You can then append route "users" instead of hard-coding.

Reply:
I didn't do an env variable but I did move the base API_URL into main so that if other DBAccess's need it we can use it like that. Let me know if you still think it's necessary to make a _true_ Environment Variable. See commit da4da39d5eb19e14f37cc1b5e3f03bbb7f81022b for details.